<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="GraphicsLayoutWidget" name="img_view">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Edit Colors</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QListWidget" name="listWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Change Selected To</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_choose_color">
           <item>
            <property name="text">
             <string>red</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>green</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>blue</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>yellow</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>magenta</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>cyan</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave_View"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load Images</string>
   </property>
  </action>
  <action name="actionLoad_2">
   <property name="text">
    <string>Load 2</string>
   </property>
  </action>
  <action name="actionLoad_3">
   <property name="text">
    <string>Load 3</string>
   </property>
  </action>
  <action name="actionLoad_4">
   <property name="text">
    <string>Load 4</string>
   </property>
  </action>
  <action name="actionLoad_5">
   <property name="text">
    <string>Load 5</string>
   </property>
  </action>
  <action name="actionLoad_6">
   <property name="text">
    <string>Load 6</string>
   </property>
  </action>
  <action name="actionSave_View">
   <property name="text">
    <string>Save View</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
